templates/join-webmaker-modal.html

```diff
@@ -28,14 +28,14 @@ <h3 class="modal-title" ng-show="currentState === MODALSTATE.welcome">
       <div class="alert alert-danger" ng-show="form.user.$error.invalidEmail" ng-bind-html="'NotAnEmail' | i18n"></div>
       <div class="terms-checkbox checkbox">
         <input id="agree-to-terms" ng-model="user.agree" type="checkbox" ng-disabled="form.user.$error.accountExists" ng-change="agreeToTermsChanged();" name="agree" tabindex="2">
-        <label for="agree-to-terms" tabindex="2">
+        <label for="agree-to-terms">
           <div><span></span></div>
           <span ng-bind-html="'webmakerAuthAgreeToTerms' | i18n"></span>
         </label>
       </div>
       <div class="mailing-list-checkbox checkbox">
-        <input id="subscribe-to-list" ng-model="user.subscribeToList" type="checkbox" ng-disabled="form.user.$error.accountExists" name="subscribeToList" tabindex="2">
-        <label for="subscribe-to-list" tabindex="2">
+        <input id="subscribe-to-list" ng-model="user.subscribeToList" type="checkbox" ng-disabled="form.user.$error.accountExists" name="subscribeToList" tabindex="3">
+        <label for="subscribe-to-list" >
```

Comment:
Can you remove the blank space after the `for` attribute and the closing `>`? There are some changes in the signin modal that need this fix too.